deepchem/feat/graph_features.py

```diff
@@ -424,3 +434,145 @@ def _featurize(self, mol):
+                   7.5, 8.0, 8.5, 9.0, 9.5, 10.0, 10.5, 11.0, 11.5, 12.0
+               ], [0.0, 4.0, 8.0], [0.4]],
+               layer_sizes=[32, 32, 16],
+               strip_hydrogens=True,
+               learning_rate=0.001,
+               epochs=10):
+    """
+    Parameters
+
+    labels: numpy.ndarray
+      Labels which we want to predict using the model
+    neighbor_cutoff: int
+      TODO (VIGS25): Add description
+    frag1_num_atoms: int
+      Number of atoms in first fragment
+    frag2_num_atoms: int
+      Number of atoms in second fragment
+    complex_num_atoms: int
+      TODO (VIGS25) : Add description
+    max_num_neighbors: int
+      Maximum number of neighbors possible for an atom
+    batch_size: int
+      Batch size used for training and evaluation
+    atom_types: list
+      List of atoms recognized by model. Atoms are indicated by their
+      nuclear numbers.
+    radial: list
+      TODO (VIGS25): Add description
+    layer_sizes: list
+      List of layer sizes for the AtomicConvolutional Network
+    strip_hydrogens: bool
+      Whether to remove hydrogens while computing neighbor features
+    learning_rate: float
+      Learning rate for training the model
+    epochs: int
+      Number of epochs to train the model for
+    """
+
+    self.atomic_conv_model = dc.models.tensorgraph.models.atomic_conv.AtomicConvModel(
+        frag1_num_atoms=frag1_num_atoms,
+        frag2_num_atoms=frag2_num_atoms,
+        complex_num_atoms=complex_num_atoms,
+        max_num_neighbors=max_num_neighbors,
+        batch_size=batch_size,
+        atom_types=atom_types,
+        radial=radial,
+        layer_sizes=layer_sizes,
+        learning_rate=learning_rate)
+
+    super(AtomicConvFeaturizer, self).__init__(
+        frag1_num_atoms=frag1_num_atoms,
+        frag2_num_atoms=frag2_num_atoms,
+        complex_num_atoms=complex_num_atoms,
+        max_num_neighbors=max_num_neighbors,
+        neighbor_cutoff=neighbor_cutoff,
+        strip_hydrogens=strip_hydrogens)
+
+    self.epochs = epochs
+    self.labels = labels
+
+  def featurize_complexes(self, mol_files, protein_files):
+    pool = multiprocessing.Pool()
+    results = []
+    for i, (mol_file, protein_pdb) in enumerate(zip(mol_files, protein_files)):
+      log_message = "Featurizing %d / %d" % (i, len(mol_files))
+      results.append(
+          pool.apply_async(_featurize_complex,
+                           (self, mol_file, protein_pdb, log_message)))
+    pool.close()
+    features = []
+    failures = []
+    for ind, result in enumerate(results):
+      new_features = result.get()
+      # Handle loading failures which return None
+      if new_features is not None:
+        features.append(new_features)
+      else:
+        failures.append(ind)
+
+    features = np.asarray(features)
+    labels = np.delete(self.labels, failures)
+    dataset = DiskDataset.from_numpy(features, labels)
+
+    # Fit atomic conv model
```

Comment:
Is the goal to use this featurizer to extract embedded feature vectors from a trained AtomicConvModel?

Reply:
Thats what I assumed, otherwise there is the `ComplexNeighborListFragmentAtomicCoordinates` featurizer already available.

Reply:
From the issue:
"The PDBBind featurization introduced in #1360 currently only supports featurization with the `RdkitGridFeaturizer`. This should be modified to use the `atomic_conv_model` in `deepchem.models.tensorgraph.atomic_conv`."

Reply:
Oh, sorry I wasn't clear in #1362. The intention of that issue was to extend `dc.molnet.load_pdbbind` to perform atomic convolutional featurization for the PDBBind dataset so users could get a featurized dataset ready for use with `AtomicConvModel`.

This current PR is pretty interesting as well. It's basically a embedder for co-crystal structures which could be neat in its own right. I'd be open to merging the current PR as-is.

Reply:
Ah, I got confused too, sorry.

It looks like that issue is then sorted, because `ComplexNeighborListFragmentAtomicCoordinates` is already the featurizer for `AtomicConvModel`. 

Yes, I thought about this too and thats why it looked interesting! Only concern is if it makes sense to have features used to predict the same task labels that the embedding network was trained on?